File: spec/dummy-bot-ruby/scraper.rb
require 'json'

$stderr.puts('hello from ruby')

puts({:n => 1, :hello => 'hello, 1'}.to_json)
puts({:n => 2, :hello => 'hello, 2'}.to_json)
puts({:n => 3}.to_json)
puts({:n => 4, :hello => 'hello, 4'}.to_json)

File: spec/dummy-bot-python/scraper.py
from __future__ import print_function

import json
import sys

print('hello from python', file=sys.stderr)

print(json.dumps({'n': 5, 'hello': 'hello, 5'}))
print(json.dumps({'n': 6, 'hello': 'hello, 6'}))
print(json.dumps({'n': 7}))
print(json.dumps({'n': 8, 'hello': 'hello, 8'}))
